dbt_copilot_helper/utils/aws.py

```diff
@@ -8,7 +8,7 @@
 SSM_PATH = "/copilot/{app}/{env}/secrets/{name}"
 
 
-def check_aws_conn(aws_profile: str) -> boto3.session.Session:
+def check_and_return_aws_session(aws_profile: str) -> boto3.session.Session:
```

Comment:
I think I'd prefer `get_aws_session` for this method as this describes the happy path for it.

Reply:
I'd agree if the function was just getting and returning the aws session, but it also aborts with messages if there's a failure. Given the side effects I prefer more detail. Perhaps `get_aws_session_or_abort`?

Reply:
I'm ok with `get_<thing>_or_abort`.

Though I'm still not a fan of function names including side effects, for instance we have `get_connection_secret_arn` which gets a connection secret, but will raise an exception (`SecretNotFoundConduitError`) if it fails, I don't think this should be called `get_connection_secret_arn_or_raise_no_secret_error`.

Reply:
I don't really like functions like this exit-ing halfway through. I'd prefer throwing an exception and handling it higher up the stack, in which case I'd be fine with get_thing. The abort isn't something you'd normally do in a utility method. I'd refactor, but it's used in quite a few places at this point.